spark/src/main/java/io/ddf/spark/SparkDDFManager.java

```diff
@@ -272,8 +272,23 @@ public DDF copyFrom(DDF ddf) throws DDFException {
       if (ddf instanceof S3DDF) {
         S3DDF s3DDF = (S3DDF) ddf;
         S3DataSourceCredentials cred = s3DDF.getManager().getCredential();
-        uri = String.format("s3a://%s:%s@%s/%s", cred.getAwsKeyID(), cred.getAwsScretKey(),
-            s3DDF.getBucket(), s3DDF.getKey());
+        if (s3DDF.getPath() != null) {
+          String[] fileList;
+          String path = s3DDF.getPath();
+          fileList = path.split(",");
```

Comment:
This wouldn't work against pattern like `adatao-sample-data/test/parquet/wildcard/{part=1,part=2}/`.
We should add minimal processing to paths.
If we want to support multiple paths, make user code explicitly pass a list of paths instead.